app/build.gradle.kts

```diff
@@ -24,7 +24,7 @@ android {
         multiDexEnabled = true
         targetSdk = 36
         versionCode = 131
```

Comment:
You also have to bump the version code at the same time, otherwise Android won't detect an actual change of the version.